lang/po/Firmware_cs.po

```diff
@@ -224,8 +224,8 @@ msgid ""
 "Calibrating XYZ. Rotate the knob to move the Z carriage up to the end "
 "stoppers. Click when done."
 msgstr ""
-"Kalibrace XYZ. Otacenim tlacitka posunte Z osu az k~hornimu dorazu. Potvrdte "
-"tlacitkem."
+"Kalibrace XYZ. Otacenim tlacitka posunte Z osu az k~hornimu dorazu. Potvrdte"
+" tlacitkem."
```

Comment:
There are some minor differences in the `.po` files with long strings. I could not fix this by adjusting the string width. So I’m not sure if this is a mistake in the `.po` file or how to resolve it if it is a mistake in `polib` side in Python.